tests/admin_views/tests.py

```diff
@@ -789,6 +789,49 @@ def test_allows_attributeerror_to_bubble_up(self):
         with self.assertRaises(AttributeError):
             self.client.get(reverse('admin:admin_views_simple_changelist'))
 
+    def test_after_add_message_contains_change_link(self):
```

Comment:
The new test seems a bit redundant with the other test changes. Does it test anything different?